### PR TITLE
Add support for OneGate wallet

### LINKED ISSUE #145: Add support for OneGate wallet
## Is your feature request related to a problem? Please describe.

I'm always frustrated when we can't work with wallets supported by https://governance.neo.org/

## Describe the solution you'd like

Research OneGate and what's needed to support it.

### PR DESCRIPTION
**Pull request by mike-petrov**
Signed-off-by: Mikhail Petrov <mike@nspcc.ru>

closes #145

### CODE REVIEW

**Comment by mike-petrov** [comment]
We have one issue: we can't open and download objects because it's not allowed in a browser type like the wallet. What can we do about it? For example, we can display images in popup and they will be visible, but other types of files are not suitable for displaying.

image

File: src/App.js [hunk 1]
```diff
@@ -482,18 +499,44 @@ export const App = () => {
...
+				wcSdk.setSession(session);
+			} catch (error) {
+				onModal('failed', 'Something went wrong, contact the application administrator');
+			}
 		}
 	}
 
 	const onDisconnectWallet = async () => {
-		await wcSdk.disconnect();
+		if (!dapi) {
```
Comment (roman-khimov) [inline src/App.js:536]: But what about `if dapi`? How can we disconnect?

File: src/Components/TreeView/TreeView.js [hunk 2]
```diff
@@ -178,106 +178,110 @@ const File = ({
...
-												window.URL.revokeObjectURL(url);
-												document.body.removeChild(a);
-											}, 0);
-										});
-									}}
-									width={40}
-									height={40}
-									alt="download an object"
-								/>
+								{!window.OneGate && (
```
Comment (roman-khimov) [inline src/Components/TreeView/TreeView.js:181]: The only thing for OneGate is `delete`? Downloads (with appropriate Content-Disposition) get blocked too?

File: src/App.js (hunk 1 above)
Comment (mike-petrov) [reply src/App.js:536]: There is no such method in the [documentation](https://github.com/neo-ngd/neo-dapi-monorepo/tree/master/packages/neo-dapi), so we just redirect the user to the main page

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**